### ISSUE TITLE
Retinanet is supported?

### ISSUE THREAD

**Issue by LaoYang1994** [facebookresearch/maskrcnn-benchmark#916]
## ❓ Questions and Help
I'm confused about the retinanet config. Does this project has supported retinanet?

**Comment by fmassa** [comment]
Yes, it is supported, but we do not provide pre-trained models for it https://github.com/facebookresearch/maskrcnn-benchmark/tree/master/configs/retinanet

**Comment by SkeletonOne** [comment]
https://github.com/facebookresearch/maskrcnn-benchmark/pull/102#issue-228098274 
You can refer to this PR
